CONTRIBUTING.md

```diff
@@ -47,12 +47,16 @@ Then, [submit a pull request][pr] on GitHub to the CHAOSS repository.
 [pr]: https://github.com/thoughtbot/factory_girl_rails/compare/
 [clone]: https://help.github.com/articles/cloning-a-repository/
 
-At this point you are waiting on us. We will comment on your pull requests
+At this point you are waiting on the CHAOSS repository maintainers. They will comment on your pull requests
 within three business days (and, typically, one business day).
 
-The CHAOSS maintainers will report on open issues and pull requests on the calls and via the mail list to elicit feedback from the community.
+The CHAOSS repository maintainers will report on open issues and pull requests on the calls and via the mail list to elicit feedback from the community.
```

Comment:
Linking calls would be useful, here.

Reply:
I added a link as suggested. Thanks @tobie